tests/cupy_tests/core_tests/test_ndarray_reduction.py

```diff
@@ -189,70 +189,70 @@ def test_min_inf(self, xp, dtype, exclude_cutensor):
     @testing.numpy_cupy_allclose(contiguous_check=False)
     def test_ptp_all(self, xp, dtype):
         a = testing.shaped_random((2, 3), xp, dtype, order=self.order)
-        return a.ptp()
+        return xp.ptp(a)
```

Comment:
The `xp.ptp` tests are already exist in `tests/cupy_tests/statistics_tests/test_order.py`. Could you remove these tests?

Reply:
Removed